client/java/README.md

```diff
@@ -0,0 +1,69 @@
+# OpenLineage Java Client
+
+Java client for [OpenLineage](https://openlineage.io).
+
+## Installation
+
+Maven:
+
+```xml
+<dependency>
+    <groupId>io.openlineage</groupId>
+    <artifactId>openlineage-java</artifactId>
+    <version>0.5.1</version>
```

Comment:
0.6 - should match what we have in different parts

Reply:
I agree, but the version in the `README.md` should represent the current version of the java client. That said, I should update the version to `0.5.1` (which is the [current](https://github.com/OpenLineage/OpenLineage/releases/tag/0.5.2) published release)